dawn/src/dawn/SIR/proto/SIR/statements.proto

```diff
@@ -622,13 +623,24 @@ message LiteralAccessExpr {
   int32 ID = 5;            // ID of the Expr
 }
 
+message Weight {
```

Comment:
A brief explanation of what a weight is would be nice here, since it's interface.